src/components/Camera/style.css

```diff
@@ -176,3 +219,46 @@
   text-decoration: underline;
   cursor: pointer;
 }
+
+.absoluteBottom() {
```

Comment:
`absoluteBottom` seems misleadingly general, this set of rules is really only fit for `.captureActions`
Is it not possible to put inside of `.captureActions`> maybe with multiple selectors as well?

Reply:
The fact that these selectors are marked with `()` will exclude them from final ouptut. 

The names used for those mixins tried to closely represent to what their real implementation is, and decouple it to what elements they will affect.